src/store/blockchain/actions.ts

```diff
@@ -91,10 +92,7 @@ export const toggleTokenLock: ThunkCreator<Promise<any>> = (token: Token, isUnlo
                 token.address,
                 ethAccount,
                 new BigNumber('0'),
-                {
-                    ...TX_DEFAULTS,
-                    gasPrice,
-                },
+                getTransactionOptions(gasPrice),
             );
         } else {
             tx = await contractWrappers.erc20Token.setUnlimitedProxyAllowanceAsync(token.address, ethAccount);
```

Comment:
Q: @mariano-aguero should we add the transaction options here too? Same question for line `161` in this file.

Reply:
Yes, you can check this [right here](https://github.com/0xProject/0x-monorepo/blob/7423028fea4970306d7280bffa603844728d1431/packages/contract-wrappers/src/contract_wrappers/erc20_token_wrapper.ts#L236)
I will added in the next commit